packages/math/base-elements.lua

```diff
@@ -795,7 +814,7 @@ function elements.underOver:_stretchyReshapeToBase (part)
       if elt:is_a(elements.text) and elt.kind == "operator" and SU.boolean(elt.stretchy, false) then
          elt:_horizStretchyReshape(self.base.width)
       end
-   elseif part:is_a(elements.underOver) then
+   elseif part:is_a(elements.underOver) and (self.attributes.accent or self.attributes.accentunder) then
```

Comment:
This condition is dubious, and I might have messed up a rebase conflict here. Under check.

Reply:
So dubious that it breaks f19 and f22 in the Mozilla Test Suite :rofl:

Reply:
Lol.